### PR TITLE
Update with 2020 values

### PR DESCRIPTION
**Pull request by rpitingolo**
@ptatian can you please review and merge if it looks ok (I'm not sure if I can merge on a forked repo). Thanks!

### CODE REVIEW

**Comment by ptatian** [comment]
@rpitingolo Yes, I will have to merge it. Looking at it now.

File: Dollar_convert.sas [hunk 1]
```diff
@@ -211,7 +213,8 @@
 	%let CPI_2016 = 223.807;  %** Annual 2016 **;
 	%let CPI_2017 = 227.241;  %** Annual 2017 **;
 	%let CPI_2018 = 231.789;  %** Annual 2018 **;  
-	%let CPI_2019 = 233.415;  %** first half average **;  
+	%let CPI_2019 = 234.970;  %** Annual 2019 **;
```
Comment (ptatian) [inline Dollar_convert.sas:216]: I see the 2019 annual average as 234.215. Would you please double check?

File: Dollar_convert.sas [hunk 2]
```diff
@@ -160,7 +161,8 @@
 	%let CPI_2016 = 240.007;  %** Annual 2016 **;
 	%let CPI_2017 = 245.120;  %** Annual 2017 **;
 	%let CPI_2018 = 251.107;  %** Annual 2018 **; 
-	%let CPI_2019 = 254.412;  %** first half average **;  
+	%let CPI_2019 = 255.658;  %** Annual 2019 **;
```
Comment (ptatian) [inline Dollar_convert.sas:164]: 2019 annual average = 255.657?

File: Dollar_convert.sas [hunk 3]
```diff
@@ -160,7 +161,8 @@
 	%let CPI_2016 = 240.007;  %** Annual 2016 **;
 	%let CPI_2017 = 245.120;  %** Annual 2017 **;
 	%let CPI_2018 = 251.107;  %** Annual 2018 **; 
-	%let CPI_2019 = 254.412;  %** first half average **;  
+	%let CPI_2019 = 255.658;  %** Annual 2019 **;  
+	%let CPI_2020 = 258.255;  %** 2020-Q1 **;
```
Comment (ptatian) [inline Dollar_convert.sas:165]: Not sure how you came up with the quarterly number. Did you average together Jan, Feb, Mar? I cannot match your value.

**Comment by rpitingolo** [comment]
@ptatian thanks for the review. I went back and updated the 2019 annual numbers. I think they are OK now. For 2020 I averaged the first three quarters of data together.

For all items its: 257.971 | 258.678 | 258.115
For less shelter its: 235.553 | 236.084 | 235.184

If there is a better way to do 2020 at this point I am happy to incorporate it. Thanks!

**Pull request merged**